r/R/csv.R

```diff
@@ -385,7 +385,13 @@ CsvTableReader$create <- function(file,
 #'
 #' `CsvReadOptions$create()` further accepts these additional arguments:
 #'
-#' - `skip_rows` Number of lines to skip before reading data (default 0)
+#' - `skip_rows` Number of lines to skip before reading data (default 0).
+#' - `skip_rows_after_names` Number of lines to skip after the column names (default 0).
+#' This number can be larger than the number of rows in one block, and empty rows are counted.
+#' The order of application is as follows:
+#'   - `skip_rows` is applied (if non-zero);
+#'   - column names aread (unless `column_names` is set);
+#'   - `skip_rows_after_names` is applied (if non-zero).
```

Comment:
```suggestion
#' - `skip_rows_after_names` Number of lines to skip after the column names (default 0).
#'    This number can be larger than the number of rows in one block, and empty rows are counted.
#'    The order of application is as follows:
#'      - `skip_rows` is applied (if non-zero);
#'      - column names are read (unless `column_names` is set);
#'      - `skip_rows_after_names` is applied (if non-zero).
```

Reply:
Thank you for finding this. This is obviously copied from the following, so the following line also needs to be corrected too.
(I will send you a PR later)

https://github.com/apache/arrow/blob/6a4bcb36c091fea07c03c57b2e31dd29f9846ac2/python/pyarrow/_csv.pyx#L113